lib/src/views/home/widgets/recipe_card.dart

```diff
@@ -0,0 +1,208 @@
+import 'package:flutter/material.dart';
+import 'package:web_provise_test/assets.dart';
+import 'package:web_provise_test/src/models/recipe.dart';
+import 'package:web_provise_test/src/utils/extensions.dart';
+import 'package:web_provise_test/src/widgets/network_image_with_fallback.dart';
+
+class RecipeCard extends StatelessWidget {
+  final Recipe recipe;
+
+  const RecipeCard({super.key, required this.recipe});
+
+  @override
+  Widget build(BuildContext context) {
+    final isLandscape =
+        MediaQuery.of(context).orientation == Orientation.landscape;
+
+    return Container(
+      padding: isLandscape
+          ? const EdgeInsets.only(bottom: 8)
+          : const EdgeInsets.only(right: 8),
+      clipBehavior: Clip.hardEdge,
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(10),
+        boxShadow: [
+          BoxShadow(
+            color: Colors.grey.withOpacity(0.3),
+            spreadRadius: 3,
+            blurRadius: 10,
+            offset: const Offset(0, 0), // changes position of shadow
+          ),
+        ],
+        color: Colors.white,
+      ),
+      child: isLandscape
+          ? _buildLandscapeLayout(context)
+          : _buildPortraitLayout(context),
```

Comment:
It's best to create a widget using class instead of helper function so we don't need to care about performance issue when state changes.

Refs:
- [Widgets vs helper methods | Decoding Flutter](https://www.youtube.com/watch?v=IOyq-eTRhvo)
- [More](https://steveos.medium.com/use-separate-widget-over-helper-method-in-flutter-heres-why-better-performace-433672eb7461)